src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameResponseStream.cs

```diff
@@ -134,14 +134,14 @@ public void Abort()
             }
         }
 
-        public void Initialize(FrameContext context)
+        public void Initialize(IFrameControl context)
         {
-            _context = context;
```

Comment:
MessageBody is dependent on a few more bits, SocketInput, RejectRequest, TakeMessageHeaders. Could add to IFrameControl?